hackathon_site/dashboard/frontend/src/components/cart/CartSummary/CartSummary.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./CartSummary.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import TitledPaper from "components/general/TitledPaper/TitledPaper";
+
+const CartSummary = ({ cartQuantity }) => (
+    <TitledPaper title="Cart Summary">
+        <Container className={styles.qty}>
+            <Typography variant="body2">Quantity</Typography>
+            <Typography variant="body2">{cartQuantity}</Typography>
+        </Container>
+        <Typography variant="body2" className={styles.msg}>
+            Your entire team’s order is here. Before you submit, refresh the page if
```

Comment:
This smart apostrophy spooks me, was it copy-pasted from somewhere?

Reply:
probably copied from the Figma, I'll update